doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -190,15 +192,40 @@ In a worst case scenario, the store time can be up to several minutes.
 This can for example happen when storing a large size replay protection list.
 It is recommended to configure the settings subsystem's internal caches to improve the performance.
 
-* The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+The Settings NVS name cache reduces the number of search loops of internal parameter identifiers, keeping them in memory.
+
+* :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE`
+
+The size of the Settings NVS name cache, :kconfig:option:`CONFIG_SETTINGS_NVS_NAME_CACHE_SIZE`, is recommended to be at least equal to the number of setting entries the device is expected to store. These entries may include entries in the RPL, whose size is given by :kconfig:option:`CONFIG_BT_MESH_CRPL`, :ref:`configuration values <ug_bt_mesh_configuring_optional_features>`, application- and network keys, application specific states etc. Using :ref:`bluetooth_mesh_sensor_server` as an example configured according to the sample's :ref:`configuration guide <bluetooth_mesh_sensor_server_conf_models>`, this has the following possible entries:
+
+* RPL entries. Since the default Networked Lighting Control (NLC) config :kconfig:option:`CONFIG_BT_MESH_NLC_PERF_DEFAULT` is used, the RPL size is 32.
+* Application key(s). Assume three are used.
+* Bound application keys. Each of the three Sensor Servers and Sensor Setup Servers has one bound application key.
+* Network key(s). Assume only one key is used.
+* Model subscription(s). Each of the three Sensor Servers and Sensor Setup Servers models subscribes to a group address.
+* Model publication information. Each of the three Sensor Server models publishes to a group address.
+* Virtual address(es). Assume these are not used.
+* Label UUIDs for virtual addressing. Assume these are not used.
+* Configuration values
+* Heartbeat publication information
+* IV index
+* Network information
+* Sequence number
+* Model data. Each of the Three Sensor Server models stores the following data:
```

Comment:
```suggestion
* RPL entries - Since the default Networked Lighting Control (NLC) configuration is used (:kconfig:option:`CONFIG_BT_MESH_NLC_PERF_DEFAULT` is set), the RPL size is 32.
* Application key(s) - Assuming three keys are used.
* Bound application keys - Each of the three Sensor Server and Sensor Setup Server models has one bound application key.
* Network key(s) - Assuming only one key is used.
* Model subscription(s) - Each of the three Sensor Server and Sensor Setup Server models subscribes to a group address.
* Model publication information - Each of the three Sensor Server models publishes to a group address.
* Virtual address(es) - Assuming these are not used.
* Label UUIDs for virtual addressing - Assuming these are not used.
* Configuration values
* Heartbeat publication information
* IV index
* Network information
* Sequence number
* Model data - Each of the three Sensor Server models stores the following data:
```